apps/meteor/lib/getMessageUrlRegex.ts

```diff
@@ -1,2 +1,2 @@
 export const getMessageUrlRegex = (): RegExp =>
-	/([A-Za-z]{3,9}):\/\/([-;:&=\+\$,\w]+@{1})?([-A-Za-z0-9\.]+)+:?(\d+)?((\/[-\+=!:~%\/\.@\,\w]*)?\??([-\+=&!:;%@\/\.\,\w]+)?(?:#([^\s\)]+))?)?/g;
+	/(https?:\/\/)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&\/=]*)/g;
```

Comment:
We do not want to force https... could be any protocol here, could you change it, please?